py3status/modules/window_title_async.py

```diff
@@ -0,0 +1,114 @@
+"""
+Display the current window title with async update.
+
+Uses asynchronous update via i3 IPC events.
+Provides instant title update only when it required.
+
+Configuration parameters:
+    - format : format of the title, default: "{title}".
+    - empty_title : string that will be shown instead of the title
+                    when the title is hidden, default: " ".
+    - always_show : do not hide the title when it can be already
+                    visible (e.g. in tabbed layout), default: False.
+
+Requires:
+    - i3ipc (https://github.com/acrisci/i3ipc-python)
+
+@author Anon1234 https://github.com/Anon1234
+@license BSD
+"""
+
+from threading import Thread
+
+import i3ipc
+
+
+class Py3status:
+
+    format = "{title}"
+    empty_title = " "
```

Comment:
please use "" instead, this will make i3bar not to show the module completely instead of having to separators separated by an empty space ;)